src/lib/i18n/texts/de.json

```diff
@@ -102,6 +107,53 @@
     "skinLocked": "Skin gesperrt",
     "requiredLevel": "Benötigtes Level",
     "weaponStrength": "Waffenstärke",
-    "currentlyEquipped": "Aktuell ausgerüstet"
+    "defense": "Verteidigung",
+    "currentlyEquipped": "Aktuell ausgerüstet",
+    "accountBound": "Accountgebunden",
+    "soulBound": "Seelengebunden"
+  },
+  "itemAttributes": {
+    "power": "Kraft",
+    "precision": "Präzision",
+    "toughness": "Zähigkeit",
+    "vitality": "Vitalität",
+    "boonDuration": "Segensdauer",
+    "conditionDamage": "Zustandsschaden",
+    "conditionDuration": "Zustandsdauer",
+    "ferocity": "Wildheit",
+    "healingPower": "Heilkraft",
+    "armor": "Rüstung",
+    "critChance": "Kritische Trefferchance",
+    "critDamage": "Kritischer Schaden",
+    "health": "Lebenspunkte",
+    "expertise": "Fachkenntnis",
+    "concentration": "Konzentration",
+    "agonyResistance": "Qual-Widerstand",
+    "magic": "Magisches Gespür",
+    "elementalist": "Klassenspezifisches Attribut",
+    "guardian": "Klassenspezifisches Attribut",
+    "warrior": "Klassenspezifisches Attribut",
+    "engineer": "Klassenspezifisches Attribut",
+    "ranger": "Klassenspezifisches Attribut",
+    "thief": "Klassenspezifisches Attribut",
+    "mesmer": "Klassenspezifisches Attribut",
+    "necromancer": "Klassenspezifisches Attribut",
+    "revanent": "Klassenspezifisches Attribut"
+  },
+  "accSummary": {
+    "raidBossKills": "Raid Boss Kills",
```

Comment:
This has been renamed to "Raid Bosses" in my current branch https://github.com/madou/armory-react/tree/feature/MoreGraphicalUserPage